### DISCUSSION TITLE
Is there a release date for OpenTelemetry.Instrumentation.XXX

### DISCUSSION THREAD

**Discussion by hejingkan2005** [open-telemetry/opentelemetry-dotnet#3815] [General]
Moving OneDS to OTel per docs, still using release candidate versions for several OpenTelemetry.Instrumentation.XXX libraries, like OpenTelemetry.Instrumentation.AspNetCore

###   **<PackageReference Update="OpenTelemetry.Instrumentation.Http" Version="1.0.0-rc9.4" />**

Any release date for OpenTelemetry.Instrumentation.XXX is appreciated.

**Comment by cijothomas** [comment]
https://github.com/open-telemetry/opentelemetry-dotnet/milestone/23
